### DISCUSSION TITLE
The top toolbar should have a fixed position

### DISCUSSION THREAD

**Discussion by jpbaril** [music-assistant/support#2806] [Feature requests and ideas]
In my opinion, the top toolbar should have a fixed position, i.e. stays visible at top even when scrolling down. That way when scrolling a long list the toolbar buttons, especially the search button and field, are always accessible.
![Capture d’écran du 2024-08-25 01-20-31](https://github.com/user-attachments/assets/9a6f5d6a-7081-45b2-8a45-6f8e7573d695)

**Comment by OzGav** [comment]
Duplicate https://github.com/orgs/music-assistant/discussions/2571
